packages/core/src/plugins/popup-controller.class.ts

```diff
@@ -1,11 +1,22 @@
 import { AuguryWindow } from '../augury-window.interface';
 
+interface WindowSizeAndPosition {
+  position: { x: number; y: number };
+  size: { width: number; height: number };
+}
+
 export class PopupController {
   public readonly window: AuguryWindow;
+  protected storageKey: string;
   private readonly onUnload: () => void;
 
   constructor(protected readonly name: string) {
     this.onUnload = this.kill.bind(this);
+    this.storageKey = `${this.name
```

Comment:
Does this get saved as per the context of the host angular application? We might want to prefix this with something that indicates it's for augury and also lessen and conflicts.

Reply:
Yeah, I could change the name to something like `augury-popup-settings`?